Makefile

```
@@ -16,8 +16,8 @@ clean:
 
 .PHONY: install
 install: $(TARGET).so
-	cp $^ $(PREFIX)
+	install -Dm644 -t $(DESTDIR)/$(PREFIX)/lib $^
```

Thanks for fixing up the Makefile!

```suggestion
	install -Dm644 -t $(DESTDIR)$(PREFIX)/lib $^
```

---

FIxed!